requirements/adapter.txt

```diff
@@ -4,6 +4,7 @@
 boto3<=2
 bottle>=0.12,<1
 chalice>=1.28,<2;
+cheroot<11                              # https://github.com/slackapi/bolt-python/issues/1374
```

Comment:
praise: Thanks for linking the issue so we know why we're requiring a lower version of `cheroot`. Let's hope this works on the CI side as well! 🤞🏻